deploy/helm/sumologic/values.yaml

```
@@ -2102,15 +2102,36 @@ otelcol:
         ## than this value.
         limit_mib: 1900
 
-      ## The queued_retry processor uses a bounded queue to relay batches from the receiver or previous
-      ## processor to the next processor.
-      queued_retry:
-        ## Number of workers that dequeue batches
-        num_workers: 16
-        ## Maximum number of batches kept in memory before data is dropped
-        queue_size: 10000
-        ## Whether to retry on failure or give up and drop
-        retry_on_failure: true
+      ## Smart cascading filtering rules with preset limits.
+      cascading_filter:
+        ## (default = 30s): Wait time since the first span of a trace before making
+        ## a filtering decision
```

Not sure if we want those comments about the defaults.

If we really want it then I'd put it at the end of the comment like so (and small suggestion regarding wording used):

```suggestion
        ## Wait time between obtaining first span of a trace and making a filtering decision
        ## Default: 30s
```

---

I would skip the defaults too
@mat-rumian

---

@sumo-drosiek I am going to update this section a little bit

---

I gave it a though and I think this is complex enough and almost always requires assigning values by user to better have too much rather too little description